app/models/person.rb

```diff
@@ -738,7 +738,11 @@ def residency_eligible?
   end
 
   def age_on(date)
-    age = date.year - dob.year
+    begin
+      age = date.year - dob.year
+    rescue
+      flash_message = { :error => "Please provide a valid birthdate."}
```

Comment:
@eidarus we need a translation for this message

Reply:
we aren't doing translations for any other of the non-client specific error messages, are we sure we want to do it here? This same error is gonna apply to every client

Reply:
if it displays to the end user, it needs to be in a translation string